model/Hierarchy.php

```diff
@@ -1,9 +1,13 @@
 <?php
 /**
- * DataObjects that use the Hierarchy extension can be be organised as a hierarchy, with children and parents.
- * The most obvious example of this is SiteTree.
+ * DataObjects that use the Hierarchy extension can be be organised as a hierarchy, with children and parents. The most
+ * obvious example of this is SiteTree.
+ *
  * @package framework
  * @subpackage model
+ *
+ * @property int ParentID
+ * @method DataObject Parent
```

Comment:
The documentation on these two extensions is to be used with the `@mixin` annotation which will mix the target class' autocomplete into the current class. If you extend a class with `Hierarchy`, then place `@mixin Hierarchy` in that class' docblock, IDEs that support `@mixin` will then autocomplete these properties as if they belonged to the extended class.